src/demo/apps/DialerApp.tsx

```diff
@@ -79,9 +79,22 @@ export function DialerApp(props: AppProps): React.ReactElement {
         <div>
             <Number inCall={inCall} number={phoneNumber} onValueChange={onNumberEntry} />
             {inCall && <CallTimer />}
-            {!inCall && <Dialer handleKeyPress={onDialerEntry} />}
+            {!inCall && <Dialer handleKeyPress={
+                (() => {
+                    if (Math.random() < 0.5) {
+                        return onDialerEntry;
+                    } else {
+                        return () => {};
+                    }
+                })()
+            } />}
```

Comment:
Can we remove this? Looks like I was testing something, forgot to clean up again :-)

Reply:
Fixed